docs/configuration/features/sequence_features.md

```diff
@@ -1,45 +1,70 @@
 ## Sequence Features Preprocessing
 
-Sequence features are transformed into an integer valued matrix of size `n x l` (where `n` is the size of the dataset and `l` is the minimum of the length of the longest sequence and a `sequence_length_limit` parameter) and added to HDF5 with a key that reflects the name of column in the dataset.
-The way sequences are mapped into integers consists in first using a tokenizer to map from strings to sequences of tokens (by default this is done by splitting on spaces).
-Then a dictionary of all the different token strings present in the column of the dataset is collected, then they are ranked by frequency and an increasing integer ID is assigned to them from the most frequent to the most rare (with 0 being assigned to `<PAD>` used for padding and 1 assigned to `<UNK>` item).
+Sequence features are transformed into an integer valued matrix of size `n x l` (where `n` is the number of rows and `l`
+is the minimum of the length of the longest sequence and a `sequence_length_limit` parameter) and added to HDF5 with a
+key that reflects the name of column in the dataset.
+The way sequences are mapped into integers consists of first using a tokenizer to map text to sequences of tokens
+(default tokenization is done by splitting on spaces).
+Next, a dictionary is constructed which maps each unique token to its frequency in the dataset column. Tokens are ranked
+by frequency and a sequential integer ID is assigned from the most frequent to the most rare (with 0 being assigned to
+`<PAD>` used for padding and 1 assigned to `<UNK>` item).
```

Comment:
"... most rare. Ludwig uses `<PAD>`, `<UNK>`, `<SOS>`, and `<EOS>` special symbols for padding, unknown, start, and end, consistent with common NLP deep learning practices. Special symbols can also be set manually in the config"

Reply:
Good catch, thanks!